release-deploy/action.yml

```diff
@@ -22,12 +23,14 @@ runs:
         zipName="*-${{ inputs.gyg_env }}-${{ inputs.gyg_region }}-${{ inputs.version }}.zip"
         msg="Downloading artifact '$zipName' for deployment"
         echo $msg
-        gh release download -p $zipName ${{ inputs.version }}
+        release_version=`echo "${{ inputs.version }}" | sed -E 's/.*(v[0-9]+\.[0-9]+\.[0-9]+).*/\1/'`
```

Comment:
Isn't the github release called `v1.2.3-some-branch`? We would want that, wouldn't we? ie. if I'm packaging from `some-branch`, then the artifact that gets published in github releases should be easily identifiable that it has come from `some-branch`, right?

Reply:
At the moment the github release is still `v1.2.3` with the artifacts being `*-v1.2.3-some-branch.zip` - I'll take a look and see if we can release the way you're thinking

Reply:
@edwardaux 
Please look at 
Release artifact with branch in version: https://github.com/guzmanygomez/bhyve-feature-example/releases
Deploying with branch in version: https://github.com/guzmanygomez/bhyve-feature-example/actions/runs/1856593294

Note the way it works it suffixes a "prerelease" version e.g. [v1.0.14-BHPL-675-deploy-branches.0](https://github.com/guzmanygomez/bhyve-feature-example/releases/tag/v1.0.14-BHPL-675-deploy-branches.0)[](https://github.com/guzmanygomez/bhyve-feature-example/releases/latest)

It does open the ability to add "prerelease" as an input to version (in addition to major|minor|patch)